### PR TITLE
Features/gilads/add prom back to public

### PR DESCRIPTION
**Pull request by giladsu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [x] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by giladsu** [comment]
Hello -  a few notes about this PR:
1. This change was already reviewed by ARM when it was originally added to the public repo in this [PR](https://github.com/Azure/azure-rest-api-specs/pull/16613).  It was moved out of the public repo to the private repo for a small amount of time, and we are now ready to remerge it to the public repo.

2. The swagger is the same except for a few description updates and small changes needed to pass validation tools.

A couple questions:

1. We would like to block documentation from being auto-generated and published for now (since this is still limited preview). Can you please advise on how to do this?

2.  This api-version is also found in the private repo - does it need to be removed or does a different api-version need to be used in the public repo? Or is it OK as is?

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+          "description": "The properties of an action group object."
+        }
+      }
+    },
+    "PrometheusRuleGroupProperties": {
+      "type": "object",
+      "required": [
+        "scopes",
+        "rules"
+      ],
```
[the remark this answers is not in the document]
Comment (yanivlavi) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json:390]: Expression is the only other field that is mandatory, so yes.

**Review by j5lim — APPROVED** [review]
This is to merge an already approved PR. Approved from ARM side.

**Comment by giladsu** [comment]
@j5lim - thank you for the signoff.

Can you advise on what can be done to prevent documentation to be autogenerated?
Also, just to make sure - it is OK that the same api version exists in the private repo?

**Comment by j5lim** [comment]
> @j5lim - thank you for the signoff.
> 
> Can you advise on what can be done to prevent documentation to be autogenerated? Also, just to make sure - it is OK that the same api version exists in the private repo?

I think those questions more related to SDK. The SDK team should be able to help.

**Comment by weidongxu-microsoft** [comment]
@giladsu

What do you mean `to prevent documentation to be autogenerated`? AFAIK msdoc on new api-version need to be configured/updated, so no automation on that part.

But I wonder why you want to have your API not documented.

**Review by weidongxu-microsoft — APPROVED** [review]
I can merge when your service is ready.

**Comment by giladsu** [comment]
Thanks @weidongxu-microsoft - we are currently only in private preview for 1P so the PMs requested that no public documentation will be generated for this resource type for now.

Initially we planned to be in private repo in order to prevent this, but since we are integrating with Azure Service Operator it needs to be in the public repo.  I will forward you a thread with some more details.

**Pull request closed without merging**